### ISSUE TITLE
buildH logo

### ISSUE THREAD

**Issue by patrickfuchs** [patrickfuchs/buildH#108]
Here's a first try for **buildH** logo. 

![buildH](https://user-images.githubusercontent.com/13885052/120687042-eb379800-c4a1-11eb-9eb8-c58ecc154bdc.png)

What do you think?

**Comment by HubLot** [comment]
I like it very much!!
It may be a bit to wide for a logo. One could replace the "Sch" by a drawing the theta angle between one CH and a Z-axis (to drawn too) in the 2nd picture.

**Comment by patrickfuchs** [comment]
Thanks @HubLot! I thought about adding theta angle, but I'd like to keep S_CH to make clear that the ultimate goal is to compute the order parameter. But I agree it was too wide 😅 Here's a second proposition:

![buildH_2](https://user-images.githubusercontent.com/13885052/120791181-28e50100-c534-11eb-9765-28336b96538b.png)

I realize the blue color for vector z may not be the best, but I wanted a color clearly different from the others. I reduced the widthness, do you think it's still too wide?

**Comment by pierrepo** [comment]
I like very much this second version, more compact than the first.

**Comment by HubLot** [comment]
same for me! For the blue color, you can try a less "flashy" blue, like [1774ac](https://www.colorhexa.com/1774ac) but anyway, it's great!

**Comment by patrickfuchs** [comment]
Thanks for your feedback ! I followed your suggestion with the blue:

![buildH_3](https://user-images.githubusercontent.com/13885052/120986137-8f5c6000-c77c-11eb-90d9-ec45ce89ecf6.png)

I think it's nice like this 😄!

**Comment by HubLot** [comment]
yes!!

**Comment by patrickfuchs** [comment]
> @patrickfuchs I think you can put this nice logo into the `docs` folder. We could then add it into the `README.md` and into the documentation.

Done in 81ce6f7. That's very nice to see the logo when reading the README or the first doc page 😝 !

Now that would be nice to add the logo on top of the left menu. @pierrepo I looked at your link, I found this paragraph that seems interesting: https://jupyterbook.org/start/overview.html#book-configuration-config-yml
But I'm not sure how to do this. Do we need to create a yaml file?
